notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<h2>Ingest feature values into entity types from a Pandas DataFrame</h2><p> </p><p>Ingest feature values into entity types that contain your features. Afterwards, you can <code>read</code>&nbsp;(online) or&nbsp;<code>batch serve</code>&nbsp;(offline) the feature values from the feature store. The following steps ingest feature values from a Pandas DataFrame into an entity type. You can also import feature values from BigQuery or Google Cloud Storage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>